tasks/db_deploy/db_deploy.py

```diff
@@ -60,11 +62,14 @@ def task(config: Dict[str, str]) -> None:
 
     # Connect as admin user to the postgres database
     with postgres_admin_engine.connect() as connection:
-        # Check if database exists, if not throw an error
+        # Check if database exists. If not, start from scratch.
         if not app_db_exists(connection):
             logger.critical("The ORCA database disaster_recovery does not exist.")
```

Comment:
Possibly change this log severity from a critical to an info or warn now and put an info logging statement after the DB is created.

Reply:
Fixed

Reply:
Add  log info statement after database is created to let end user know the database was created.

Reply:
Added